doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -277,36 +206,36 @@ Linear Algebra Equivalents
 
    * - ``[ 1 2 3; 4 5 6 ]``
      - ``array([[1.,2.,3.], [4.,5.,6.]])``
-     - 2x3 matrix literal
+     - 2D array defined as 2x3 matrix
```

Comment:
After you pointed it out, I don't like it either. I'm replacing `matrix` with `2D array`, since there are places that sometimes refer to matrix as `2D matrix` and other times just `matrix`. 

Both MATLAB and NumPy deal with arrays, so I think its best to keep the `array` verbage.